Sources/Sentry/SentryTracer.m

```diff
@@ -195,32 +195,36 @@ - (void)dispatchIdleTimeout
 {
     @synchronized(_idleTimeoutLock) {
         if (_idleTimeoutBlock != NULL) {
-            [_configuration.dispatchQueueWrapper dispatchCancel:_idleTimeoutBlock];
+            [SentryDependencyContainer.sharedInstance.dispatchQueueWrapper
```

Comment:
`h`: I would either add a `dispatchQueueWrapper` property to the SentryTracer and set it to `SentryDependencyContainer.sharedInstance.dispatchQueueWrapper` or ask for it via the init methods of the tracer cause it's a required dependency. I prefer the second one for testability as we once aligned [here](https://github.com/getsentry/sentry-cocoa/blob/main/develop-docs/DECISIONS.md#dependency-injection-strategy), but I would also be OK with the first option.

Reply:
I would approve a PR if you opened one showing me what you mean though

Reply:
I opened a PR https://github.com/getsentry/sentry-cocoa/pull/3497.

Reply:
Merged that!